en/components/grid-column-pinning.md

```diff
@@ -1,20 +1,24 @@
 ---
 title: Grid Column Pinning - Grid Feature
-_description: The Grid Column Pinning is a mechanism to pin the first few columns of a scrollable Grid. 
+_description: The Grid Column Pinning is a mechanism to pin selected columns of a scrollable Grid.
 _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
 ---
 
 ## Grid Column Pinning
 
-Use the Grid Column Pinning Component to fix the first few columns of a scrollable Grid to the left. Now, when the user scrolls, all other columns move left and right except for the pinned ones. The pinned columns always appear on top of the scrollable ones. The Grid Column Pinning is visually identical to the [Ignite UI for Angular Grid Column Pinning Feature](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid_column_pinning.html)
+Use the Grid Column Pinning as a mechanism to fix the first few columns of a scrollable Grid to the left. Now, when the user scrolls, all other columns move left and right except for the pinned ones. The pinned columns always appear on top of the scrollable ones. The Grid Column Pinning is visually identical to the [Ignite UI for Angular Grid Column Pinning Feature](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid_column_pinning.html)
 
 ### Grid Column Pinning Demo
 
 <img class="responsive-img" src="../images/grid_column_pinning_demo.png" srcset="../images/grid_column_pinning_demo@2x.png 2x" />
 
+### Header Cell Feature
+
+In order to pin a column use either the `Feature Left` or `Feature Right` overrides and set it to `Pinning`. Of course, you would probably also want to rearrange the column order so that the pinned ones come first when looking left to right.
+
 ### Cell Right Border
 
-The Grid Column Pinning is achieved by changing the Right Border override to Pinned Line for the Header Cell and all Body Cells that constitute the right-most column of the pinned ones. In the demo above, this is done for all cells of the second column, resulting in the pinning of the two columns to the left.
+The right-most pinned column should have its `Right Border` override to indicate `Pinned Line` both for the Header and all Body Cells. In the demo above, this is done for all cells of the second column since both the Row Selectors and the Subject column are pinned.
```

Comment:
It is included:  **In the demo above**, this is done for all cells of the second column since both the Row Selectors and the Subject column are pinned.

but there is no more information below that paragraph.

Reply:
Added the image once more